examples/Example/src/prot_reanimated/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { GestureMonitor, Gesture } from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { useAnimatedGesture } from '../useAnimatedGesture';
+import { useRef } from 'react';
+
+function Draggable() {
+  const [a, sa] = useState(1);
+
+  useEffect(() => {
+    setInterval(() => {
+      sa((a) => a + 1);
+    }, 1000);
+  }, []);
+
+  const pressed = useSharedValue(false);
+  const offsetX = useSharedValue(0);
+  const offsetY = useSharedValue(0);
```

Comment:
AFAIK you can use objects there:
```ts
const offset = useSharedValue({x: 0, y: 0});
```

Reply:
Fixed in 6d23678bc9c88e2bd47b7c90b14bf1feccbc0b45